### PR TITLE
Update information about the DiscordServerStatus plugin and prerequisites

### PR DESCRIPTION
**Pull request by Buff-oG**
Explained and informed how to tests DiscordServerStatus plugin can be tested and works.

### CODE REVIEW

**Review by werewolfboy13 — CHANGES_REQUESTED** [review]
[REJECTED] Cron jobs will overwrite this, needs to be added to https://github.com/Team-Silver-Sphere/SquadJS/blob/31aad28f7e369d7e46ce40f607252c16c9d4e7d8/squad-server/plugins/discord-server-status.js#L10 for it to work.

**Comment by Buff-oG** [comment]
Is it okay after I edited the readme template, discord-server-status JS file stating with a Description of how to use the plugin and package json new version ID.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "SquadJS",
-  "version": "3.7.0",
+  "version": "3.7.1",
```
Comment (werewolfboy13) [inline package.json]: Please do not bump the versioning, that is done internally.

File: squad-server/plugins/discord-server-status.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default class DiscordServerStatus extends DiscordBaseMessageUpdater {
       ...DiscordBaseMessageUpdater.optionsSpecification,
       command: {
         required: false,
-        description: 'Command name to get message.',
+        description: 'Command name to get message. Discord bot requires "Message Content Intent" option to be checked via Discord Developer Porta. Otherwise your bot cannot read the chat/channel',
```
Comment (werewolfboy13) [inline squad-server/plugins/discord-server-status.js]: Typo

**Review by werewolfboy13 — DISMISSED** [review]

File: squad-server/plugins/discord-server-status.js (hunk 2 above)
Comment (Buff-oG) [reply squad-server/plugins/discord-server-status.js]: Woups...

**Comment by Buff-oG** [comment]
Conversions done. Typo fixed.

**Comment by werewolfboy13** [comment]
Unable to push into this PR, will need access @Buff-oG

**Pull request closed without merging**